### PR TITLE
Epic 7 - Yana Lebedynska

### PR DESCRIPTION
**Pull request by yanalebedynska**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/yana_lebedynska/Epic7/practice.cpp [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    {
+        newMatrix[i] = new int[10];
+    }
+
+    // Заповнення матриці значеннями згідно вказаного шаблону
+    for (int i = 0; i < 10; i++)
+    {
+        for (int l = 0; l < 10; l++)
+        {
+            newMatrix[i][l] = number + i * 10 + l;
```
Comment (fm2lpnu) [inline ai_12/yana_lebedynska/Epic7/practice.cpp:169]: Task 1 (a,b) is not implemented according to requirements.

File: ai_12/yana_lebedynska/Epic7/practice.cpp [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    cout << "Matrix before sorting: " << endl;
+    PrintMatrix(newMatrix);
+
+    // Зчитування значення для обробки введеного користувачем елемента
+    cout << "Enter the pull request element: ";
+    int PR;
+    cin >> PR;
+
+    // Обробка матриці та запис результату у файл
+    Processing(newMatrix, PR);
```
Comment (fm2lpnu) [inline ai_12/yana_lebedynska/Epic7/practice.cpp:183]: Task 2 (a,b) is not implemented according to requirements.
Function is not overloaded.

File: ai_12/yana_lebedynska/Epic7/practice.cpp [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+        for (int l = 0; l < 10; l++)
+        {
+            cout << matrix[i][l] << " ";
+        }
+        cout << endl;
+    }
+}
+
+// Функція для обробки матриці та запису результату у файл
+bool Processing(int** matrix, int element)
```
Comment (fm2lpnu) [inline ai_12/yana_lebedynska/Epic7/practice.cpp:82]: Not following the requirements.
`Processing` body had to be implemented by multiple functions and data had to be passed between them.

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**